tests/integration/project/elements/sandbox-bwrap/test-dev-shm.bst

```diff
@@ -0,0 +1,15 @@
+kind: manual
```

Comment:
It looks like this test does not actually test the new sandbox flag.

Ideally we should update the base image to use a new glibc and have a `BuildElement` plugin which exercises the new flag.

Let me see if I can update this quickly and get this merged.

Reply:
Update, after discussing this on slack we will update this patch to also add yaml configuration for this for `BuildElement` and `ScriptElement` classes so that projects can easily enable creation of `/dev/shm`, since this is more urgent with glibc 2.34 as highlighted in #1525